pkg/utils/db.go

```diff
@@ -278,17 +279,109 @@ func IsErrBinlogPurged(err error) bool {
 	return ok && e.Code == tmysql.ErrMasterFatalErrorReadingBinlog
 }
 
-// IsErrTableNotExists checks whether err is TableNotExists error
-func IsErrTableNotExists(err error) bool {
-	return IsMySQLError(err, tmysql.ErrNoSuchTable)
+// IsNoSuchThreadError checks whether err is NoSuchThreadError
+func IsNoSuchThreadError(err error) bool {
+	return IsMySQLError(err, tmysql.ErrNoSuchThread)
 }
 
-// IsErrDupEntry checks whether err is DupEntry error
-func IsErrDupEntry(err error) bool {
-	return IsMySQLError(err, tmysql.ErrDupEntry)
+// BaseConn Connect to Database
```

Comment:
one unimportant question, which one is better?

1. add a new pkg to support base DB connection operations
2. keep the base DB connection operations in this `utils` pkg